networks/moonriver.md

```diff
@@ -15,10 +15,10 @@ In order to collect as much feedback as possible and provide fast issue resoluti
 
 ## Initial Configurations
 
-Moonriver is scheduled to follow a [5-phase launch process](https://moonbeam.network/networks/moonriver/launch/). Currently, Moonriver is in Phase 0 of the launch process and has the following configurations:
```

Comment:
what about creating a Phase Number variable so you can change it in 1 place :)

Reply:
I don't think it's worth it to put this one thing in a variable, as we do not reuse it, and I know to keep track of it